### ISSUE TITLE
Can i run on SSD

### ISSUE THREAD

**Issue by civa** [node-real/bsc-erigon#145]
I know syncing BSC on ssd doesnt work. is anyone running erigon on SSD? ill appreciate your feedback

**Comment by 0x1un** [comment]
I running erigon bsc archive node on lvm disk ( two nvme ssd disk).
At the moment, it always seems to be tens of thousands of blocks behind, because it is very slow in the execution phase, usually processing only 3-4 blocks a second.

**Comment by delid4ve** [comment]
@0x1un did you sync from scratch or using the snapshots? I've tried the snapshots multiple times now but they just don't extract - always EOF errors as others report too.

**Comment by 0x1un** [comment]
> @0x1un did you sync from scratch or using the snapshots? I've tried the snapshots multiple times now but they just don't extract - always EOF errors as others report too.

yes, my node is synchronized from snapshot. That snapshot is only the execute stage section and does not contain the download snapshots stage section

**Comment by calmbeing** [comment]
Suggest NVME or IO2 SSD
